### ISSUE TITLE
VSR option in list

### ISSUE THREAD

**Issue by DimulyaPlay** [Orbmu2k/nvidiaProfileInspector#219]
I tryin to make shortcut to enable/disable vsr, because it steal too much fps from game when video plays on other monitor. Switching it in NCP every time I want to play so stupid. Chat gpt recommended this util to make "vsr on" and "vsr off" profiles. I can set it from two batch files (or one). But I cant find any VSR option in options list... Is it in one of "unknown" options?

**Comment by BlueFllame** [comment]
I agree that having this option visible would be a good idea. I also think they should try to include everything from the legacy control panel and the new app so this way we'll never need to install any Nvidia non-driver software.
